modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenStrain.C

```diff
@@ -40,8 +40,7 @@ ComputeThermalExpansionEigenStrain::computeQpStressFreeStrain()
 
   if (!_has_incremental_strain || _step_one)
     old_temp = _stress_free_reference_temperature;
-
```

Comment:
This is heading the right way, but I think we should make more changes to this. I think this should behave the way the solid_mechanics version does, although others may disagree. `stress_free_reference_temperature` should be called `stress_free_temperature` to avoid confusion with the reference temperature, which is something else, and which I plan to add shortly. It also shouldn't have a default in my opinion.